bin/test-storybook.js

```diff
@@ -158,6 +163,45 @@ async function checkStorybook(url) {
   }
 }
 
+// Copied from github.com/nrwl/nx/blob/1975181c200eb288221c8beb94e268fe9659cc26/packages/nx/src/utils/package-manager.ts#L29-35
+function detectPackageManager(dir = '') {
+  return fs.existsSync(path.join(dir, 'pnpm-lock.yaml'))
+    ? 'pnpm'
+    : fs.existsSync(path.join(dir, 'yarn.lock'))
+    ? 'yarn'
+    : 'npm';
+}
+
+// Copied from github.com/nrwl/nx/blob/1975181c200eb288221c8beb94e268fe9659cc26/packages/nx/src/utils/package-manager.ts#L48-106
+function getPackageManagerCommand(packageManager = detectPackageManager()) {
```

Comment:
What if people are running this in a monorepo that only has a package.json at the root level? If they're using yarn workspaces, for instance, this function will endup saying that it's an `npm` project, right?

Reply:
All scripts are usually run with cwd set to same directory as the package.json (eg. `npm run ...` automatically takes care of that even if you run it from a directory within your porject).
We are using a nx mono repository with one single package.json in the root of the repo, tested it and works fine